templates/CMakeLists.txt.template

```diff
@@ -295,57 +298,53 @@
   add_custom_target(tools
     DEPENDS tools_c tools_cxx)
 
-  if (gRPC_BUILD_TESTS)
-  add_custom_target(buildtests_c)
-  % for tgt in targets:
-  % if tgt.build == 'test' and not tgt.language == 'c++' and not tgt.get('external_deps', None) and not tgt.boringssl:
-  ${get_platforms_condition_begin(tgt.platforms)}\
-  add_dependencies(buildtests_c ${tgt.name})
-  ${get_platforms_condition_end(tgt.platforms)}\
-  % endif
-  % endfor
+  if(gRPC_BUILD_TESTS)
+    add_custom_target(buildtests_c)
+    % for tgt in targets:
+    % if tgt.build == 'test' and not tgt.language == 'c++' and not tgt.get('external_deps', None) and not tgt.boringssl:
+    ${platforms_condition_add_dependencies(tgt, 'buildtests_c')}
```

Comment:
nit: this does `add_dependencies('buildtests_c',  tgt.name)` , but  the parameter order of platforms_condition_add_dependencies is the reverse`tgt, 'buildtests_c'`, that seems confusing

Reply:
No longer relevant, as this function has been removed in the latest version.